opm/parser/eclipse/EclipseState/Tables/Tables.cpp

```diff
@@ -67,6 +67,7 @@
 #include <opm/parser/eclipse/EclipseState/Tables/SwofTable.hpp>
 #include <opm/parser/eclipse/EclipseState/Tables/TableContainer.hpp>
 #include <opm/parser/eclipse/EclipseState/Tables/WatvisctTable.hpp>
+#include <opm/parser/eclipse/EclipseState/Tables/DensityTable.hpp>
```

Comment:
is DENSITY a table keyword? As far as I can remember, there's only one entry per record and nothing is interpolated?